bibliopixel/project/project.py

```diff
@@ -62,7 +64,11 @@ def make_remote(layout, remote):
         run = anim.get('run', {})
         run['threaded'] = True  # Remote anims must be threaded, maybe a little hacky?
         anim_obj = dict(anim)
-        anim_obj['animation'] = make_animation(layout, anim['animation'], run)
+        try:
+            anim_obj['animation'] = make_animation(layout, anim['animation'], run)
+        except:
+            log.error(traceback.format_exc())
+            anim_obj['animation'] = None
```

Comment:
How about instead I have anim_obj['loaded'] = False for things that bombed? It's more verbose, but it solves that problem. It also solves another quibble you had up above inside control.py